src/server/wesnothd/game.hpp

```diff
@@ -15,11 +15,11 @@
 
 #pragma once
 
+#include "string_enums/side_controller.hpp"
```

Comment:
Should sort later alphabetically, now that it's not called enum_maps.

Sorry for such a trivial comment, but small fixes are good.

Reply:
Done.